packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -80,4 +80,9 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Notifies when tooltip becomes visible or hidden, whatever the trigger was.
    */
   onTooltipToggle?(isTooltipVisible: boolean): void;
+
+  /**
+   * Optional function to enable customization (such as defer-loading) on display of tooltip callout
+   */
+  onTooltipRender?(tooltip: ITooltipProps): void;
```

Comment:
You'll want to add an example to the existing `TooltipPage` in the documentation. This will also help you find and fix bugs.